### DISCUSSION TITLE
FingerPaint v1.2.2

### DISCUSSION THREAD

**Discussion by Wazzaps** [Wazzaps/fingerpaint#10] [Announcements]
Work with old `pillow` versions (i.e. the one that comes with Ubuntu 20.04)

<hr /><em>This discussion was created from the release <a href='https://github.com/Wazzaps/fingerpaint/releases/tag/v1.2.2'>FingerPaint v1.2.2</a>.</em>

**Comment by Wazzaps** [comment]
@jhossbach Could you bump the AUR version?

**Reply by jhossbach** [reply]
Done :+1:
